src/custom-element.js

```diff
@@ -1617,7 +1617,9 @@ function createBaseCustomElementClass(win) {
       }
 
       if (
-        this.layoutCount_ > 0 ||
+        ((this.isFirstLayoutCompleted_ ||
```

Comment:
We use `layoutCount` when to indicate both successful and unsuccessful layout. So this would allow failed layouts to show a loading indicator.

Reply:
Ok, I'll switch back to layoutCount from `isFirstLayoutCompleted_` then